make/photon/prepare/scripts/gencert.sh

```diff
@@ -0,0 +1,118 @@
+#! /bin/bash
+
+# CA key and certificate
+openssl req -x509 -nodes -days 365 -newkey rsa:4096 \
```

Comment:
It would be better if the script has flexibility to allow user to cert his CA.

But I think that can be another PR.